config/iostability_test.yaml

```diff
@@ -16,3 +16,9 @@ degraded_path_durations_days: 7
 
 test_40172:
   bucket_creation_healthy_mode: True
+
+test_40041:
+  nearfull_memory_percetange: 95
+
+test_40173:
+  nearfull_memory_percetange: 95
```

Comment:
Do you need to create two different fields for same value? I guess one 'nearfull_memory_percetange' should serve the requirement.

Reply:
Can you change the word memory to storage?

Reply:
updated with only one key value pair